### PR TITLE
Introduced yml file (GitHub Action) for fastjson release 

### PR DESCRIPTION
**Pull request by jobyko**
Release script will 

1. Prepare the jar file. 
2. Later archive jar with requried set of files 
3. Attach the same to the release page.

Requried set of changes to be to be introduced 
1. Fix the build failure for Windows/ Mac OS.
2. Intermittent failure in tag publish in the GitHub

@asa1997  @Pranshu021 
Kindly perform a critical review

@sumodgeorge

### CODE REVIEW

**Comment by asa1997** [comment]
@jobyko 
- This pr has to be raised under the branch where the patch has to be released.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Create Release
+
+on:
+  push:
+    # Sequence of  patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to  matching v*, i.e. v1.0, v20.15.10
```
Comment (asa1997) [inline .github/workflows/release.yml:7]: The tag has to match our requirements.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@jobyko Please address the comments

File: .github/workflows/release.yml (hunk 1 above)
Comment (jobyko) [reply .github/workflows/release.yml:7]: @asa1997 
Based on requirement the tag name will be like, 1.2.24-bes0.1-CVE-2017-18349

Will change it.

**Comment by jobyko** [comment]
Closing the PR to include the changes requested.

**Pull request closed without merging**